STYLE.md

```diff
@@ -103,38 +103,65 @@ Attributes exported by your modules should follow the same descriptive format as
 
 Only comment what is necessary.
 
-Single line comments: `#` or `//`.
+Single line comments: `#` .
 
 Multi-line comments: `/*` followed by `*/`.
 
 When commenting use two "//" and a space in front of the comment.
 
-```
-// CREATE ELK IAM ROLE 
+```hcl
+# CREATE ELK IAM ROLE 
 ...
 ```
 
 ## Naming Conventions
 
-### File Names
+### File Names and Structure
+
+We strongly recommend that terraform modules and resources are grouped in a feature-based structure, this means that you should always strive to start with main.tf and split your resources into features, where each file groups a set of resources.
 
-Create a separate resource file for each type of AWS resource. Similar resources should be defined in the same file and named accordingly.
+If your module is less complex, please attempt to flatten the directory structure
 
+```shell
+$ tree minimal-module/
```

Comment:
I think examples should _always_ be included. Could just leave that as a note under the directory structure, and "see the complex module structure for the layout of examples" or something.

Reply:
@rickardl - This comment was on the previous commit, but I think it's still valid?